charts/argo-cd/templates/argocd-applicationset/clusterrolebinding.yaml

```diff
@@ -0,0 +1,17 @@
+{{- $config := .Values.controller.clusterAdminAccess | default dict -}}
```

Comment:
Please remove the backward compat from the old param of the application-controller. It doesn't make sense to add something from the other controller (application-controller vs. applicationSET-controller) here.